src/CarbonAware.CLI/src/Commands/Emissions/EmissionsCommand.cs

```diff
@@ -13,25 +18,50 @@ class EmissionsCommand : Command
             new string[] { "--start-time", "-s" },
             LocalizableStrings.StartTimeDescription)
             {
-                IsRequired = false,
-                Arity = ArgumentArity.ExactlyOne,
+                Arity = ArgumentArity.ZeroOrOne,
             };
     private readonly Option<DateTimeOffset?> _endTime = new Option<DateTimeOffset?>(
              new string[] { "--end-time", "-e" },
             LocalizableStrings.EndTimeDescription)
             {
-                IsRequired = false,
-                Arity = ArgumentArity.ExactlyOne,
+                Arity = ArgumentArity.ZeroOrOne,
+            };
+    private readonly Option<bool> _best = new Option<bool>(
+            new string[] { "--best", "-b" },
+            LocalizableStrings.BestDescription)
+            {
+                Arity = ArgumentArity.ZeroOrOne,
+            };
+    private readonly Option<bool> _average = new Option<bool>(
+            new string[] { "--average", "-a" },
+           LocalizableStrings.AverageDescription)
+            {
+                Arity = ArgumentArity.ZeroOrOne,
             };
     public EmissionsCommand() : base("emissions", LocalizableStrings.EmissionsCommandDescription)
     {
         AddOption(_requiredLocation);
         AddOption(_startTime);
         AddOption(_endTime);
+        AddOption(_best);
+        AddOption(_average);
+
+        AddValidator(ValidateOptions);
+
         this.SetHandler(this.Run);
     }
 
-    private async Task Run(InvocationContext context)
+    private void ValidateOptions(CommandResult commandResult)
```

Comment:
Rather than having the comment below, could we rename this method to be self-describing?

Reply:
I kept the method name generic so that in the future we could potentially add more validations apart from mutually exclusive options. But I guess, I am ok with renaming it to what it does today.